### DISCUSSION TITLE
What does mean [WARNING DRT-0225] o_ctrl_scan_test_mode 402 pin not visited, fall back to feedthrough mode?

### DISCUSSION THREAD

**Discussion by titan73** [The-OpenROAD-Project/OpenROAD#5714] [Q&A] [answered]
What could be the reason for not being visited.
402 pin => 402 pins?
What is feedthrough mode?

**Comment by maliberty** [comment]
@osamahammad21 please respond

**Comment by titan73** [comment]
Oh btw, o_ctrl_scan_test_mode is a net in this message, right? Well originally it's a port but I suppose the net as the same name.

**Comment by titan73** [comment, marked as answer]
Answered in https://github.com/The-OpenROAD-Project/OpenROAD/issues/5554.
